packages/admin-next/dashboard/src/v2-routes/locations/location-detail/location-detail.tsx

```diff
@@ -0,0 +1,39 @@
+import { Outlet, json, useParams } from "react-router-dom"
+
+import { JsonViewSection } from "../../../components/common/json-view-section"
+import { LocationGeneralSection } from "../../../modules/locations/location-detail/components/location-general-section"
+import { LocationSalesChannelSection } from "../../../modules/locations/location-detail/components/location-sales-channel-section"
+import { useAdminStockLocations } from "medusa-react"
+
+export const LocationDetail = () => {
+  const { id } = useParams()
+  const { stock_locations, isLoading, isError, error } = useAdminStockLocations(
+    {
+      id,
+      expand: "address,sales_channels",
```

Comment:
todo: shouldn't we use the new API?

Reply:
the api's for stock locations are not changed, this hits the new api endpoint :) or do you mean using `fields` and not `expand`?

Reply:
Yes 😅